templates/remove_approved_nodes.cdc

```diff
@@ -17,7 +17,9 @@ transaction(ids: [String]) {
     }
 
     execute {
-		let nodeIDs = FlowIDTableStaking.getApprovedList()
+
+        let nodeIDs = FlowIDTableStaking.getApprovedList()
```

Comment:
to match whats in [Flow core contract](https://github.com/onflow/flow-core-contracts/blob/5696ec5e3e6aa5fc10762cbfeb42b9c5c0b8ddbe/transactions/idTableStaking/admin/remove_approved_nodes.cdc#L15-L16)